weblate/templates/translation.html

```diff
@@ -185,6 +185,9 @@ <h4 class="panel-title">
 <div class="panel panel-default">
   <div class="panel-heading"><h4 class="panel-title">{% trans "Other components" %}</h4></div>
 {% include "snippets/list-objects.html" with objects=other_translations name_source="component_name" label=_("Component") in_panel=True %}
+  <div class="panel-footer">
+    <a href="{% url 'project-language' lang=object.language.code project=object.component.project.slug %}">See all components</a>
```

Comment:
Please make the text localizable. Also in similar context we use "Browse all ...", so please make the text consistent.

Reply:
Oops, my bad. I just changed the text to be more consistent and wrapped it with trans tag. 
Thank you for reviewing!